contracts/contracts/interfaces/IAllowList.sol

```diff
@@ -2,6 +2,11 @@
 pragma solidity ^0.8.0;
 
 interface IAllowList {
+  event AdminAdded(address indexed sender, address indexed account);
+  event EnabledAdded(address indexed sender, address indexed account);
+  event ManagerAdded(address indexed sender, address indexed account);
+  event RoleRemoved(address indexed sender, address indexed account);
```

Comment:
I think the event should include this information (and in this order of priority with respect to whether it's indexed).

1. modified account
2. new role
3. account that caused the modification
4. previous role

Since that's 4 fields, one of them would need to be captured in the function name or we'd need to make at least one non-indexed.

I'd lean towards making it a single event and leaving the previous role un-indexed since it's unlikely imo a user would need to search all of the allow list modifications that changed the status from manager -> X.

Reply:
done